spec/models/widget_spec.rb

```diff
@@ -285,6 +285,20 @@
     end
   end
 
+  describe ".paper_trail.update_columns", versioning: true do
+    it "creates a version record" do
+      widget = Widget.create
+      expect(widget.versions.count).to eq(1)
+      Timecop.travel 1.second.since # because MySQL lacks fractional seconds precision
+      widget.paper_trail.update_columns(name: "Bugle")
+      # widget.update_attributes(name: "Bugle")
+      expect(widget.versions.count).to eq(2)
+      expect(widget.versions.last.event).to(eq("update"))
+      expect(widget.versions.last.changeset[:name]).to eq([nil, "Bugle"])
+      expect(widget.versions.last.created_at.to_i).to eq(Time.now.to_i)
```

Comment:
We can't guarantee that this expectation will pass. The operation could take more than a second, or cross a second-boundary.

Reply:
True... should I just use a `Timecop.freeze` to keep `Time.now` fixed? (That's what I did.)

Looks like there's at least one other expectation with the same problem:
```
        expect(widget.paper_trail.version_at(Time.now)).to be_nil
```